src/components/Crypto/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {CryptoTypes} from '../../types';
+import {
+  CryptoContainer,
+  NameContainer,
+  Price,
+  Logo,
+  Increase,
+  Decrease,
+  Name,
+  Slug,
+  Arrow,
+  Arrowdown,
+  Arrowup,
+  Percentage,
+} from '../Crypto/styles';
+
+const Crypto = ({name, slug, price, market_porcentage, image}: CryptoTypes) => {
+  return (
+    <CryptoContainer>
+      <Logo source={image} />
+      <NameContainer>
+        <Name>{name}</Name>
+        <Slug>{slug}</Slug>
+      </NameContainer>
+      <Price>${price}</Price>
+      {parseInt(market_porcentage) > 0.0 ? (
+        <Percentage>
+          <Increase>{market_porcentage}%</Increase>
+        </Percentage>
+      ) : (
+        <Percentage>
+          <Decrease>{market_porcentage}%</Decrease>
+        </Percentage>
+      )}
+      {parseInt(market_porcentage) > 0.0 ? (
```

Comment:
> 0

Reply:
also code redundance ,use: 

```

 <Arrow>
  {parseInt(market_porcentage) > 0.0 ? (
          <Arrowup source={require('../../assets/Arrowup.png')} />
      ) : (
          <Arrowdown source={require('../../assets/Arrowdown.png')} />
      )}
      </Arrow>
```

and use camelCase in the names: ArrowDown ArrowUp